benchmark/Streamly/Benchmark/Data/Array/Unboxed/Mut.hs

```diff
@@ -140,27 +136,21 @@ composeN n f x =
         4 -> f x >>= f >>= f >>= f
         _ -> undefined
 
-scanls' :: Monad m => (b -> a -> b) -> b -> Stream.Stream m a -> Stream.Stream m b
-scanls' step z m = Stream.fromStreamD $ D.scanl' step z (Stream.toStreamD m)
-
-scanls1' :: Monad m => (a -> a -> a) -> Stream.Stream m a -> Stream.Stream m a
-scanls1' f m =  Stream.fromStreamD $ D.scanl1' f (Stream.toStreamD m)
-
 {-# INLINE scanl' #-}
 scanl' :: MonadIO m => Int -> Int -> Stream Int -> m (Stream Int)
-scanl' value n = composeN n $ onArray value $ scanls' (+) 0
+scanl' value n = composeN n $ onArray value $ Stream.scanl' (+) 0
 
 {-# INLINE scanl1' #-}
 scanl1' :: MonadIO m => Int -> Int -> Stream Int -> m (Stream Int)
-scanl1' value n = composeN n $ onArray value $ scanls1' (+)
+scanl1' value n = composeN n $ onArray value $ Stream.scanl1' (+)
 
 {-# INLINE map #-}
 map :: MonadIO m => Int -> Int -> Stream Int -> m (Stream Int)
-map value n = composeN n $ onArray value $ Prelude.fmap (+ 1)
+map value n = composeN n $ onArray value $ Stream.map (+ 1)
 
 {-# INLINE onArray #-}
 onArray
-    :: MonadIO m => Int -> (Stream.Stream m Int -> Stream.Stream m Int)
+    :: MonadIO m => Int -> (Stream.SerialT m Int -> Stream.SerialT m Int)
```

Comment:
This change and other similar changes should not be required since `Stream` and `SerialT` are aliases.